projects/hslayers/src/components/add-data/url/arcgis/add-data-url-arcgis.component.html

```diff
@@ -1,107 +1,6 @@
-                </div>
-                <input class="form-control" name="path" [(ngModel)]="data.path" />
-            </div>
-        </div>
-
-        <div class="form-group">
-            <div class="capabilities_input">
-                <label class="capabilities_label control-label"> <input type="checkbox" class="checkbox-lg"
-                        name="registerMetadata" [(ngModel)]="data.registerMetadata" /> {{'ADDLAYERS.registerMetadata' |
-                    translate}}</label>
-            </div>
-        </div>
-        <div class="d-flex flex-column my-3">
-            <div class="d-flex flex-row justify-content-start align-items-baseline mb-1">
-                {{'ADDDATA.CATALOGUE.addAs' | translate}}
-                <div class="btn-group ml-2">
-                    <button type="button" class="btn btn-sm btn-light  btn-outline-secondary" (click)="data.base = true"
-                        [ngClass]="{'active':data.base}">
-                        {{'COMPOSITONKEYWORDS.Basemap' | translate}}
-                    </button>
-                    <button type="button" class="btn btn-sm btn-light  btn-outline-secondary"
-                        (click)="data.base = false" [ngClass]="{'active': !data.base}">
-                        {{'COMMON.layer' | translate}}
-                    </button>
-                </div>
-            </div>
-            <div [hidden]="!data.base" class="input-group">
-                <div class="input-group-prepend">
-                    <span class="input-group-text control-label">{{'COMMON.title' | translate}}</span>
-                </div>
-                <input class="form-control" name="baseTitle" [(ngModel)]="data.title" />
-            </div>
-        </div>
-        <hr />
-        <table class="table table-sm table-striped table-bordered">
-            <thead>
-                <th style="width:1em">&nbsp;</th>
-                <th>{{'COMMON.name' | translate}}</th>
-            </thead>
-            <tbody *ngFor="let service of data.services">
-                <tr>
-                    <td><input class="checkbox-lg" type="checkbox" name="service" (change)="changed()"
-                            [(ngModel)]="service.checked" /></td>
-                    <td>{{service.name}}</td>
-                </tr>
-            </tbody>
-        </table>
-        <div class="w-100 d-flex justify-content-between bg-light pb-2" style="position: sticky;
-        bottom: 0">
-            <button class="btn btn-primary" (click)="selectAllLayers(data.services)">{{'ADDLAYERS.selectAllLayers' |
-                translate}}</button>
-            <button class="btn btn-primary" [disabled]="!hasChecked" [title]="'ADDLAYERS.addSelected' | translate"
-                (click)="addLayers(true)">{{'ADDLAYERS.WFS.addToMap' | translate}}</button>
-        </div>
-    </div>
+    <hs-common-url-loading-data [loadingInfo]="hsAddDataArcGisService.loadingInfo"></hs-common-url-loading-data>
```

Comment:
Could we rename to something shorter? hs-add-data-progress and hs-add-data-details maybe?